internal/components/inspektorgadget/const.go

```diff
@@ -43,6 +43,7 @@ const (
 	inspektorGadgetChartRelease   = "gadget"
 	inspektorGadgetChartNamespace = "gadget"
 	inspektorGadgetChartURL       = "oci://ghcr.io/inspektor-gadget/inspektor-gadget/charts/gadget"
+	inspektorGadgetReleaseURL     = "https://api.github.com/repos/inspektor-gadget/inspektor-gadget/releases/latest"
```

Comment:
Does Github token required to access this API?

Reply:
No. Since inpektor-gadet is public this works without the token!

Reply:
That's great!